packages/renderer/src/lib/editor/MonacoEditor.svelte

```diff
@@ -13,15 +13,35 @@ let divEl: HTMLDivElement;
 let editor: monaco.editor.IStandaloneCodeEditor;
 let Monaco;
 let isDarkUnsubscribe: Unsubscriber;
-let isDarkTheme = false;
 
 export let content = '';
 export let language = 'json';
 export let readOnly = true;
 
 const dispatch = createEventDispatcher<{ contentChange: string }>();
 
+async function updateTheme(isDarkTheme: boolean) {
+  Monaco = await import('monaco-editor');
+  // check if we're in light or dark mode and get the terminal background color
+  const appearanceUtil = new AppearanceUtil();
+  const bgColor = appearanceUtil.getColor('--pd-terminal-background');
+
+  // create a theme with the current light or dark mode, but customize the background color
+  Monaco.editor.defineTheme('podmanDesktopTheme', {
+    base: isDarkTheme ? 'vs-dark' : 'vs',
+    inherit: true,
+    rules: [{ token: 'custom-color', background: bgColor }],
+    colors: {
+      'editor.background': bgColor,
+    },
+  });
+}
+
 onMount(async () => {
+  isDarkUnsubscribe = isDark.subscribe(value => {
+    updateTheme(value);
```

Comment:
On my system, the terminal theme is not updated when I change the system theme, even if I way a few minutes.

I realize that `isDark` emits events when the configuration changes, but not when the system theme changes

Reply:
I suppose this can be part of another PR
Approving these changes